src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerConfigOptionsDictionary.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.CodeAnalysis.Diagnostics
+{
+    internal sealed class AnalyzerConfigOptionsDictionary : AnalyzerConfigOptions
```

Comment:
DictionaryAnalyzerConfigOptions? Since the naming here makes me think this is something that's a Dictionary when it's really not.